packages/web/src/data/queries/singletons/privacy.ts

```diff
@@ -0,0 +1,14 @@
+import groq from 'groq'
+
+import { RICH_TEXT } from '../objects/richText'
+
+import { META } from '../objects/meta'
+
+export const PRIVACY_PAGE = groq`
+    faq[] {
+        ${RICH_TEXT}
+    },
+    meta {
```

Comment:
meta doesn't need to be included because it's injected in `fetchDocument`

Reply:
oh yes, the `LAYOUT`, right?

Reply:
I actually got this wrong, we do need it added as it's isolated to the document.